### ISSUE TITLE
Error on example?

### ISSUE THREAD

**Issue by thearabbit** [hitchcott/app-dump#14]
I try run your example 

``` js
TypeError: Cannot call method 'split' of undefined at [object Object].route.action (packages/hitchcott:app-dump/app-dump-server.coffee:61:32) at boundNext (packages/iron:middleware-stack/lib/middleware_stack.js:251:1) at runWithEnvironment (packages/meteor/dynamics_nodejs.js:108:1) at packages/meteor/dynamics_nodejs.js:121:1 at [object Object].urlencodedParser (C:\Users\Theara\AppData\Local\.meteor\packages\iron_router\1.0.9\npm\node_modules\body-parser\lib\types\urlencoded.js:84:40) at packages/iron:router/lib/router.js:277:1 at [object Object]._.extend.withValue (packages/meteor/dynamics_nodejs.js:56:1) at [object Object].hookWithOptions (packages/iron:router/lib/router.js:276:1) at boundNext (packages/iron:middleware-stack/lib/middleware_stack.js:251:1) at runWithEnvironment (packages/meteor/dynamics_nodejs.js:108:1)
```

**Comment by thearabbit** [comment]
I base on `Windows`

**Comment by IstoraMandiri** [comment]
Sorry; I just pushed an update! Try the new version?

**Comment by thearabbit** [comment]
I tried the new, but it still problem.
I waiting you :+1: 
Now my project need this.

**Comment by Adiel-Sharabi** [comment]
1+

**Comment by msand** [comment]
I've made a pull request fixing this on windows. https://github.com/hitchcott/app-dump/pull/15

**Comment by thearabbit** [comment]
Look great :+1:

**Comment by IstoraMandiri** [comment]
Great work -- thanks for this @msand .

I'll merge test and publish this today.

**Comment by thearabbit** [comment]
Could I try now?

**Comment by IstoraMandiri** [comment]
I'll post an update once it's published @thearabbit

**Comment by thearabbit** [comment]
I am checking on `atmospherejs`.

**Comment by IstoraMandiri** [comment]
Merged PR #15 and released under `0.3.4`.

Seems to be working on windows + osx
